Detectors/TOF/reconstruction/src/Clusterer.cxx

```diff
@@ -197,7 +212,21 @@ void Clusterer::buildCluster(Cluster& c, MCLabelContainer const* digitMCTruth)
   Geo::getVolumeIndices(c.getMainContributingChannel(), det);
   float pos[3];
   Geo::getPos(det, pos);
```

Comment:
I see that Geo::getPos does a lot of job, like opening/closing a text file (debugging? Is this needed?) and query to TGeo with matrix copying etc. Why don't you cache matrices, like e.g. ITS, MFT and TRD are doing? Then you will not need TGeo at all.

Reply:
@noferini, @chiarazampolli  : the checks are green now, will you address this question in this PR or prefer to merge it at have another one?

Cheers,
 Ruben

Reply:
Hello Ruben,
I am not an expert in Geo, but I believe that one can remove the writing out of this file (at least to my understanding, and also trying the clusterization without it). But since @noferini  is the author of this part, I would let him comment. 
As for the caching, we can of course do this. Do you have an example at hand? I guess that you mean to keep in memory the matrices that have already been accessed. 

As for the merging, we can merge already now if you don't consider this a blocker. For sure we will add it in our to-do list.

Chiara

Reply:
Hi @chiarazampolli , @noferini 

OK, let's address this comment in a separate PR.
For the caching: in fact, the TGeoMatrix is supposed to be used only for TGeo building, detectors which need to correct their data (clusters) for the alignment should use faster ``o2::Transform3D `` matrices cached in the beginning (extracted from the TGeoMatrix of TGeo or, eventually, from the CCDB, since the caching class ``o2::detectors::DetMatrixCache`` (from which the detector interface to the geometry derives for ITS,MFT and TRD) is persistent. 
You can see how it is done for ``ITSMFT/ITS/base/include/ITSBase/GeometryTGeo.h`` or ``TRD/base/include/TRDBase/TRDGeometry.h``.

Cheers,
 Ruben

Reply:
For sure we can cache the matrixes. We already did something like that for other puropused.
If you agree we will address all these things in another PR.
Cheers
Francesco